docs/Course Wiki/ECE Course Offerings/ECE120.md

```diff
@@ -26,9 +26,9 @@ NOTE: While electrical engineering students normally take ECE120 in the spring t
 
-The course has weekly homeworks that are graded based on completion rather than correctness, which encourages students to at least attempt the homeworks and try their best on every question. There are then corrections submitted for questions marked wrong by graders, which make up the remaining points for the homework category. the course also has weekly labs, which are generally asynchronous programming labs in C and later LC-3 assembly, however there are also 3 breadboarding labs where students will use digital TTL chips to create digital circuits on a breadboard. Weekly discussion sections led by TAs are held every Thursday, in which students work together on a worksheet of problems to help better their understanding of topics covered that week in class. Three midterm exams and a cumulative final conclude the examination of the course. There are not very many practice exams on offer, so it is recommended to keep up with the homework for each week in order to maintain a proper understanding of the covered concepts.
+The course has weekly homeworks that are graded based on correctness, although students have a chance to submit corrections to earn up to 50% of their lost credit. There are then corrections submitted for questions marked wrong by graders, which make up the remaining points for the homework category. the course also has weekly labs, which are generally asynchronous programming labs in C and later LC-3 assembly, however there are also 3 breadboarding labs where students will use digital TTL chips to create digital circuits on a breadboard. Weekly discussion sections led by TAs are held every Thursday, in which students work together on a worksheet of problems to help better their understanding of topics covered that week in class. Three midterm exams and a cumulative final conclude the examination of the course. There are not very many practice exams on offer, so it is recommended to keep up with the homework for each week in order to maintain a proper understanding of the covered concepts.
```

Comment:
Capitalize "the" in "the course also..."
Change "on offer" to "offered"
Explain what "TTL" stands for
"Three midterm exams and cumulative final conclude the examination of the course." -> something like "Finally, to assess students' understanding of the concepts, ECE120 has three midterm exams and a cumulative final exam."